app/controllers/application_controller.rb

```diff
@@ -86,4 +89,13 @@ def set_publication
   def set_language_from_publication
     I18n.locale = @publication.language if @publication.language
   end
+
+  def breadcrumbs
+    return {} if @navigation_helpers.nil?
+    if should_present_new_navigation_view?
+      @navigation_helpers.taxon_breadcrumbs
+    else
+      @navigation_helpers.breadcrumbs
+    end
+  end
```

Comment:
That sounds like a good idea. However, that gem would have to know about the A/B testing API. Maybe @tijmenb has an idea on this?